graphene_django/rest_framework/tests/test_mutation.py

```diff
@@ -98,6 +98,25 @@ class Meta:
     assert "created" not in MyMutation.Input._meta.fields
 
 
+def test_model_serializer_required_fields():
+    class MyMutation(SerializerMutation):
+        class Meta:
+            serializer_class = MyModelSerializer
+
+    assert "cool_name" in MyMutation.Input._meta.fields
+    assert MyMutation.Input._meta.fields["cool_name"].type == NonNull(String)
+
+
+def test_model_serializer_optional_fields():
+    class MyMutation(SerializerMutation):
+        class Meta:
+            serializer_class = MyModelSerializer
+            optional_fields = "cool_name"
```

Comment:
Isn't this suppose to be a tuple or list? ie `("cool_name",)` ; It looks like this currently works because it's doing a substring search, "cool_name" is in "cool_name".

Reply:
Good catch! At first, I wrongly set `optional_fields = ('cool_name')`, then black formatted changed it to resulting `optional_fields = "cool_name"`.

Thanks for noticing, fixed.